src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CreateEmbeddingServiceCommandResolver.java

```diff
@@ -0,0 +1,55 @@
+package io.stargate.sgv2.jsonapi.service.resolver.model.impl;
+
+import io.stargate.sgv2.api.common.StargateRequestInfo;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import io.stargate.sgv2.jsonapi.api.model.command.impl.CreateEmbeddingServiceCommand;
+import io.stargate.sgv2.jsonapi.exception.ErrorCode;
+import io.stargate.sgv2.jsonapi.exception.JsonApiException;
+import io.stargate.sgv2.jsonapi.service.embedding.configuration.EmbeddingServiceConfigStore;
+import io.stargate.sgv2.jsonapi.service.embedding.configuration.PropertyBasedEmbeddingServiceConfig;
+import io.stargate.sgv2.jsonapi.service.operation.model.Operation;
+import io.stargate.sgv2.jsonapi.service.operation.model.impl.CreateEmbeddingServiceOperation;
+import io.stargate.sgv2.jsonapi.service.resolver.model.CommandResolver;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Instance;
+import jakarta.inject.Inject;
+import java.util.Optional;
+
+@ApplicationScoped
+public class CreateEmbeddingServiceCommandResolver
+    implements CommandResolver<CreateEmbeddingServiceCommand> {
+
+  private EmbeddingServiceConfigStore embeddingServiceConfigStore;
+  private StargateRequestInfo stargateRequestInfo;
+
+  @Inject
+  public CreateEmbeddingServiceCommandResolver(
+      Instance<EmbeddingServiceConfigStore> embeddingServiceConfigStore,
```

Comment:
Why is `Instance` used here? Are embedding service config stores created asynchronously? (or perhaps there's cyclic dependencies somewhere)

Reply:
Embedding Service store is created using @Produces based on the config property stargate.jsonap.embedding.config.store